### PR TITLE
#7: Tweak the main page

### PR DESCRIPTION
**Pull request by kcmikee**
## Description

This PR introduces the initial setup for the homepage layout and design updates. After discussions with team members, we've decided on a clean and minimalistic approach that aligns with our app's simplicity and functionality. The focus is on a lightweight and user-friendly interface, maintaining fast load times.

Changes Implemented:
Updated the homepage structure with a more organised layout.
Ensured the design is responsive and maintains consistency across different devices.

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes Issue: https://github.com/BuidlGuidl/batch10.buidlguidl.com/issues/7 (Tweak The Home Page) 

Your ENS/address: 0x8Bb563DC969cC3b3e7275474c3847b44dBCB9441

https://github.com/user-attachments/assets/258672b4-6a6a-4e8d-83c9-f754725c4dba

### CODE REVIEW

File: packages/nextjs/app/page.tsx
Comment (phipsae) [inline packages/nextjs/app/page.tsx:1]: In light mode, the text is hard to read. 

Screenshot 2024-11-21 at 16 08 54

File: packages/nextjs/components/StarsBg.tsx
Comment (phipsae) [inline packages/nextjs/components/StarsBg.tsx:1]: Resizing the window throws the following error in the console. 
The error you're seeing is likely related to trying to access the canvas context after the component has been unmounted or when the canvas reference is null.

Screenshot 2024-11-21 at 16 10 08

**Review by phipsae — CHANGES_REQUESTED** [review]
Very very nice landing page, thanks a lot! I really love it.

Just a little of changes and we can merge this beautiful merge. 

Thanks!

File: packages/nextjs/components/StarsBg.tsx
Comment (kcmikee) [reply packages/nextjs/components/StarsBg.tsx:1]: Noted, I'll look into it

**Comment by phipsae** [comment]
Very nice work! It's merged :)

**Pull request merged**